src/commands.ts

```diff
@@ -235,6 +236,24 @@ export class CosmosDBCommands {
         }
     }
 
+    public static async createDocDBDocument(coll: DocDBCollectionNode, explorer: CosmosDBExplorer) {
+        const masterKey = coll.db.getPrimaryMasterKey();
+        const endpoint = coll.db.getEndpoint();
+        const client = new DocumentClient(endpoint, { masterKey: masterKey });
+        await new Promise((resolve, reject) => {
+            client.createDocument(coll.getCollLink(), { 'id': 'temporary' }, (err, result) => {
```

Comment:
Using a catch-all 'temporary' word here. This would fail if the user creates a doc twice without changing the 1st doc's ID. 

Any thoughts on how to get around this?